package_test.go

```diff
@@ -671,6 +695,13 @@ func (s *PackageSuite) TestValidGetAll(c *C) {
 		inputs:   []any{},
 		slices:   []any{&[]sqlair.M{}, &[]CustomMap{}},
 		expected: []any{&[]sqlair.M{{"name": "Mark"}}, &[]CustomMap{{"id": int64(20)}}},
+	}, {
+		summary:  "simple in",
+		query:    "SELECT * AS &Person.* FROM person WHERE id IN ($S[:])",
+		types:    []any{Person{}, sqlair.S{}},
+		inputs:   []any{sqlair.S{20, 35, 36, 37, 38, 39, 40}},
+		slices:   []any{&[]*Person{}},
+		expected: []any{&[]*Person{&Person{20, "Mark", 1500}, &Person{40, "Mary", 3500}, &Person{35, "James", 4500}}},
```

Comment:
Is this test essentially duplicated?

Reply:
Yea, I can removed it. I had added it because this is `GetAll` rather than `Get` and there have been some uncaught bugs in the past because I didn't test on `GetAll`. But those were with output types and since we are only dealing here with input types that should be ok.

Reply:
Keep it then